### PR TITLE
enhancement: Environment Reconciliation

### PR DESCRIPTION
**Pull request by amsks**
Changes: 
- DMC 
- Brax

Environments checked and reconciled for newer changes from the train.

### CODE REVIEW

File: .gitmodules
Comment (TheEimer) [inline .gitmodules:1]: That is definitely a wrong change, no? Why include the workshop paper but remove mario? Can we still install it without the submodules?

File: CITATION.bib
Comment (TheEimer) [inline CITATION.bib:1]: Deleting this is also bad :D

File: README.md
Comment (TheEimer) [inline README.md:1]: The spaces are okay, but cite us should not be gone!

File: carl/envs/___carl_brax_env.py
Comment (TheEimer) [inline carl/envs/___carl_brax_env.py:1]: I'm fairly sure this can be deleted

File: carl/envs/__init__.py
Comment (TheEimer) [inline carl/envs/__init__.py:1]: We should aim to rename the brax dir to brax again

File: carl/envs/carl_env.py
Comment (TheEimer) [inline carl/envs/carl_env.py:1]: Did we change the hide context default on purpose @benjamc ?

File: carl/envs/mario/pcg_smb_env/toadgan/generate_sample.py
Comment (TheEimer) [inline carl/envs/mario/pcg_smb_env/toadgan/generate_sample.py:1]: None of these should be in here, please re-add the submodule instead. Why are they commited to main?

File: docs/source/api/index.rst
Comment (TheEimer) [inline docs/source/api/index.rst:1]: Why are the docs deleted? Re-add please!

File: experiments/Readme.md
Comment (TheEimer) [inline experiments/Readme.md:1]: The experiments were explicitly NOT supposed to be on this branch, please remove

File: notes/run_context_efficiency.sh
Comment (TheEimer) [inline notes/run_context_efficiency.sh:1]: These notes should also be removed

File: requirements_clone.txt
Comment (TheEimer) [inline requirements_clone.txt:1]: delete

**Review by TheEimer — CHANGES_REQUESTED** [review]
Please see my comments and fix

File: CITATION.bib
Comment (amsks) [reply CITATION.bib:1]: Readded the old citation. However, which citation do we want to retain? CARL or Contextualize me?

File: .gitmodules
Comment (amsks) [reply .gitmodules:1]: Readded

File: README.md
Comment (amsks) [reply README.md:1]: Readded

File: carl/envs/__init__.py
Comment (amsks) [reply carl/envs/__init__.py:1]: Changed

File: carl/envs/___carl_brax_env.py
Comment (amsks) [reply carl/envs/___carl_brax_env.py:1]: Done

File: requirements_clone.txt
Comment (amsks) [reply requirements_clone.txt:1]: Done

File: notes/run_context_efficiency.sh
Comment (amsks) [reply notes/run_context_efficiency.sh:1]: Removed

File: docs/source/api/index.rst
Comment (amsks) [reply docs/source/api/index.rst:1]: readded

File: carl/envs/carl_env.py
Comment (amsks) [reply carl/envs/carl_env.py:1]: Not sure about thi, @benjaminc?

File: carl/envs/mario/pcg_smb_env/toadgan/generate_sample.py
Comment (amsks) [reply carl/envs/mario/pcg_smb_env/toadgan/generate_sample.py:1]: readded submodules

File: CITATION.bib
Comment (TheEimer) [reply CITATION.bib:1]: Contextualize Me

**Comment by TheEimer** [comment]
I'll merge this with failing tests since we're overhauling everything in #89 anyway

**Comment by TheEimer** [comment]
Aditya, that's the wrong branch! Please open a PR on development!

**Pull request merged**